Testscripts/Linux/SetupRDMA.sh

```diff
@@ -56,32 +57,42 @@ function Main() {
 	update_repos
 	# Install common packages
 	install_package "gcc git make zip"
+	LogMsg "$?: Installed the common required packages, gcc git make zip"
```

Comment:
in method install_package, it has check_exit_status, if install package failed, it will exit the script, here $? will be always 0

Reply:
fixed in 66ae117079544b049a7a20e2d236a40d7b8e8825